packages/headless/doc-parser/use-cases/insight.ts

```diff
@@ -108,6 +108,9 @@ const actionLoaders: ActionLoaderConfiguration[] = [
   {
     initializer: 'loadCaseContextActions',
   },
+  {
+    initializer: 'loadGenericAnalyticsActions',
+  },
```

Comment:
I just noticed that when I added more action loaders to `insight.index.ts` last time, I forgot to add them here. Could add them in this PR?

Reply:
yes just added them! :)